lib/generators/aws_record/model/model_generator.rb

```diff
@@ -1,13 +1,56 @@
 # frozen_string_literal: true
 
-require_relative '../base'
+require 'rails/generators'
+
+require_relative 'generated_attribute'
+require_relative 'secondary_index'
 
 module AwsRecord
   module Generators
-    class ModelGenerator < Base
+    # Generates an Aws::Record model and table configuration.
+    class ModelGenerator < Rails::Generators::NamedBase
```

Comment:
I think I'm still missing something - where does this get required from?

Reply:
It's loaded automatically because of its subclass. Just like the session store generators.